internal/integration/config/docker-compose.yaml

```
@@ -8,7 +8,7 @@ services:
   
   postgres:
     restart: 'always'
-    image: 'postgres:15'
+    image: 'postgres'
```

is it not better to be explicit?

```suggestion
    image: 'postgres:latest'
```